cola/widgets/diff.py

```diff
@@ -745,8 +753,19 @@ def show_line_numbers(self):
         """Return True if we should show line numbers"""
         return get(self.options.show_line_numbers)
 
+    def is_enable_word_wrapping_checked(self):
+        return get(self.options.enable_word_wrap)
+
     def update_options(self):
         self.numbers.setVisible(self.show_line_numbers())
+        if self.is_enable_word_wrapping_checked():
+            self.setWordWrapMode(QtGui.QTextOption.WordWrap)
+            self.setLineWrapMode(QtWidgets.QPlainTextEdit.WidgetWidth)
+            self.options.enable_word_wrap.setChecked(True)
+        else:
+            self.setWordWrapMode(QtGui.QTextOption.NoWrap)
+            self.setLineWrapMode(QtWidgets.QPlainTextEdit.NoWrap)
```

Comment:
Please move the Qt `self.set{Word,Line}WrapMode` calls into a standalone `def set_word_wrap_enabled(self, enabled):` method and call it from here.. maybe as `self.set_word_wrap_enabled(self.is_enable_word_wrapping_check())`

It's kinda funky that we check the checkbox and then set the checkbox to the same value.. will have to see why that's setup that way.

This should probably handle the `apply_state()` / `export_state()` callbacks so that this preference persists across application restarts. eg. if you disable it, it should stay disabled when restarting cola.